app/src/main/res/drawable/ic_fire_black_24dp.xml

```diff
@@ -0,0 +1,25 @@
+<!--
```

Comment:
The naming of resources is all over the place, we have names with dp and names without. I'd like a task to clean those up but in a separate project.

Reply:
We add the color when the color is explicitly hardcoded rather than dynamic e.g if an icon is explicitly green and should only be used as a green icon then we include the color in the name. That's not the case here and since the icon is not always black it's a better to drop `black` from the name.